### ISSUE TITLE
Some tips on reducing power consumption

### ISSUE THREAD

**Issue by sukeshak** [robjordan/gpslogger-esp32#27]
Thought I will mention some more tips on reducing power consumption.

Instead of logging every 5 seconds, you can also use the distance as a trigger for logging.
In case the person stops for break, there is no point in logging GPS coordinates even if its every 5 seconds.

Interesting project... I am working on something in similar lines... Good luck!

**Comment by robjordan** [comment]
Hi Ashok

Thanks for your feedback. It's a good idea about the distance trigger.
Also, I understand that some commercial GPS loggers incorporate an
accelerometer to sleep the device when it's stationary.

Best regards
Rob

**Comment by sukeshak** [comment]
Yes. In my case its for motorsports so I have a 9 axis sensor already. 
Having accelerometer and using the interrupt to wake up also is another good idea but with additional component + cost involved.
